packages/medusa/src/services/new-totals.ts

```diff
@@ -33,6 +33,13 @@ type LineItemTotals = {
   discount_total: number
```

Comment:
todo: Do not forget to update the totals service as well for the places that are still relying on the old totals service